README.md

```diff
@@ -1 +1,52 @@
-# grid-contrib
\ No newline at end of file
+# grid-contrib
+
+The grid-contrib repository contains contributed software for
+[Hyperledger Grid](https://grid.hyperledger.org), a framework for building
+supply chain solutions for blockchain platforms.
+
+This repository is intended for community-contributed software, including
+example middleware tools and reference implementations for smart contracts
+(also called "transaction families"). This software is designed to work with
+the Hyperledger Grid platform software in the [grid
+repository](https://github.com/hyperledger/grid).
+
+For example, the Grid Track and Trace transaction family provides an off-chain
+smart contract that tracks goods as they move through a supply chain. For more
+information, see the [Grid Track and Trace
+documentation](https://grid.hyperledger.org/docs/grid/nightly/master/family_specification.html)
+and the
+[grid/track_and_trace](https://github.com/hyperledger/grid-contrib/tree/master/track_and_trace)
+subdirectory in this repository.
+
+
+## How to Contribute
+
+We welcome contributions to this repository, either as changes to existing
+software or as new example tools and reference implementations. Please follow
+the [guidelines for
+contributing](https://grid.hyperledger.org/community/contributing/). For more
+information, please join the [#grid discussion
+channel](https://chat.hyperledger.org/channel/grid) or [Hyperledger Grid
+mailing list](https://lists.hyperledger.org/g/grid).
+
+
+## More Information
```

Comment:
mention the contribs Monday meeting here, maybe?

Reply:
We should put that at "Join the Discussion" link. "Contributors Meeting" != "Meeting about contrib repo."